src/Button.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+
+export function Button(props) {
+    return (
+        <a href={props.to}>
+            {props.name}
+        </a>
```

Comment:
There's a <button> element that can be used here.  It also has a corresponding bulma class.